### PR TITLE
Removed duplicate code

### PR DESCRIPTION
**Pull request by teemuvesala**
I removed all _<method>_request methods and implemented single _common_request -method. I also removed the deprecated key words. If they're still needed, they should be wrappers to <method>_request rather than duplicating the code from <method>_request. I also did some indent fixes based to pylint. 

This change removes extra code, and makes maintaining easier. Also if you have to do some own extension for your SUT, then this simplifies also that. Overriding only one single method is easier than overriding multiple _<method>_request methods.

### CODE REVIEW

**Comment by bulkan** [comment]
@teemuvesala seems like this branch is now out of date

**Comment by teemuvesala** [comment]
> @teemuvesala seems like this branch is now out of date

I allocated from my calendar time to fix those during next Tuesday. The "Cookie edit" merge changed the functions which I edited also.

**Comment by lucagiove** [comment]
@bulkan but do you think the approach of this PR is correct?
In case I can have a look into fixing conflicts.

File: src/RequestsLibrary/RequestsKeywords.py [hunk 1]
```diff
@@ -373,9 +373,20 @@ def create_digest_session(self, alias, url, auth, headers={}, cookies=None,
             debug,
             disable_warnings)
 
-    def create_client_cert_session(self, alias, url, headers={}, cookies=None,
-                       client_certs=None, timeout=None, proxies=None,
-                       verify=False, debug=0, max_retries=3, backoff_factor=0.10, disable_warnings=0):
+    def     create_client_cert_session(
```
Comment (bulkan) [inline src/RequestsLibrary/RequestsKeywords.py]: whats happening with the spaces after `def` ? 
```suggestion
    def create_client_cert_session(....
```

File: src/RequestsLibrary/RequestsKeywords.py [hunk 2]
```diff
@@ -479,48 +490,23 @@ def get_request(
         session = self._cache.switch(alias)
         redir = True if allow_redirects is None else allow_redirects
 
-        response = self._get_request(
-            session, uri, params, headers, json, redir, timeout)
+        #response = self._get_request(
```
Comment (lucagiove) [inline src/RequestsLibrary/RequestsKeywords.py]: I think this comment could be removed

File: src/RequestsLibrary/RequestsKeywords.py [hunk 3]
```diff
@@ -981,168 +782,33 @@ def options_request(
...
-            allow_redirects,
-            timeout):
+            **args
+        ):
         self._capture_output()
-
-        resp = session.get(self._get_url(session, uri),
-                           headers=headers,
-                           json=json,
-                           params=self._utf8_urlencode(params),
```
Comment (lucagiove) [inline src/RequestsLibrary/RequestsKeywords.py]: seems that you removed some function that were executed before sending data to requests.

File: src/RequestsLibrary/RequestsKeywords.py [hunk 4]
```diff
@@ -981,168 +782,33 @@ def options_request(
...
-                           allow_redirects=allow_redirects,
-                           timeout=self._get_timeout(timeout),
```
Comment (lucagiove) [inline src/RequestsLibrary/RequestsKeywords.py]: Also session timeout seems to get lost with this change

**Comment by lucagiove** [comment]
Merged conflicts and fixed open points in the review so far #227

**Pull request merged**